src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.props

```diff
@@ -0,0 +1,59 @@
+<Project>
+  <PropertyGroup>
+    <!-- Crossgen is not used for Mono -->
+    <SkipBuild Condition="'$(RuntimeFlavor)' == 'Mono'">true</SkipBuild>
+    <PlatformPackageType>ToolPack</PlatformPackageType>
+    <SharedFrameworkName>$(SharedFrameworkName).Crossgen2</SharedFrameworkName>
+    <OverridePackageId>$(SharedFrameworkName).$(RuntimeIdentifier)</OverridePackageId>
+    <ArchiveName>dotnet-crossgen2</ArchiveName>
+    <SharedFrameworkHostFileNameOverride>crossgen2</SharedFrameworkHostFileNameOverride>
+    <GenerateInstallers>false</GenerateInstallers>
+    <PermitDllAndExeFilesLackingFileVersion>true</PermitDllAndExeFilesLackingFileVersion>
+    <!-- Publishing as single-file or NativeAOT means we can't examine the interior DLLs -->
+    <ShouldVerifyClosure>false</ShouldVerifyClosure>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference
+      Include="$(CrossgenPublishProject)"
+      OutputItemType="_RawCrossgenPublishFiles"
+      ReferenceOutputAssembly="false" />
+  </ItemGroup>
+
+  <Target Name="PublishCrossgen"
+          BeforeTargets="GetFilesToPackage">
+    <ItemGroup>
+      <_CrossgenPublishFiles Include="@(_RawCrossgenPublishFiles->'%(OutputPath)')"
+                             KeepMetadata="REMOVE_ALL" />
+    </ItemGroup>
+    <ItemGroup Condition="'$(NativeAotSupported)' != 'true'">
+      <FilesToPackage Include="@(_CrossgenPublishFiles)"
+                      Exclude="*.pdb;*.h;*.lib"
+                      TargetPath="tools/" />
+    </ItemGroup>
+    <ItemGroup Condition="'$(NativeAotSupported)' == 'true'">
+      <!-- Treat all native aot assets as native runtime assets -->
+      <FilesToPackage Include="@(_CrossgenPublishFiles->Distinct())"
+                          Condition="'%(Extension)' != '.pdb'"
+                          TargetPath="tools/" />
+    </ItemGroup>
+  </Target>
+
+  <PropertyGroup>
+    <TargetOSComponent>unix</TargetOSComponent>
```

Comment:
Just to confirm, we're not distinguishing here between Linux, MacOS, etc?

Reply:
Correct. Even better, it looks like we can remove this (this dates back to when we were shipping crossgen2 v5/v6 and is unused since then).